pkg/scheduler/api/resource_info_test.go

```diff
@@ -278,7 +278,7 @@ func TestLessEqual(t *testing.T) {
 				ScalarResources: map[v1.ResourceName]float64{"scalar.test/scalar1": 1},
 			},
 			resource2: &Resource{},
-			expected:  false,
+			expected:  true,
```

Comment:
`resource1 < resource2` in this case?

Reply:
yes, resource1.ScalarResources map is not nil, but value <= minMilliScalarResources, so continue, point 4

Reply:
how about the others, e.g. cpu, memory? we need to make sure all resources meet `LessEqual`.